### PR TITLE
Suman

### PR DESCRIPTION
**Pull request by chandan-g-bhagat**

### CODE REVIEW

File: Basic/a.html [hunk 1]
```diff
@@ -4,7 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<<<<<<< HEAD
```
Comment (khadka-suman) [inline Basic/a.html:7]: resolve this conflict

Comment (sujan-d-shrestha) [inline Basic/a.html:7]: Resolve this conflit @khadka-suman

File: Basic/profile.html [hunk 2]
```diff
@@ -2,11 +2,20 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+<<<<<<< HEAD
     <meta http-equiv="X-UA-Compatible" content="IE=edge">N
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Hey There AAAA</title>
 </head>
 <body>
+=======
```
Comment (khadka-suman) [inline Basic/profile.html:11]: resolve this conflict

File: Basic/profile.html [hunk 3]
```diff
@@ -2,11 +2,20 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+<<<<<<< HEAD
```
Comment (sujan-d-shrestha) [inline Basic/profile.html:5]: remove this conflict @khadka-suman

**Review by sujan-d-shrestha — CHANGES_REQUESTED** [review]

**Review by khadka-suman — CHANGES_REQUESTED** [review]

**Review by sujan-d-shrestha — APPROVED** [review]

**Review by khadka-suman — APPROVED** [review]

**Pull request merged**